### ISSUE TITLE
bipsea fails to start when installed with pip / running inside venv

### ISSUE THREAD

**Issue by malpka** [akarve/bipsea#64]
Environment:
Windows 11, Python 3.10.11 (in %path%), Command Prompt

Steps to reproduce:
- Start command prompt and execute:
```
python -m venv venv-bipsea
cd venv-bipsea
Scripts\activate.bat
pip install bipsea
```
```
bipsea --help
```

Results:
```
(venv-bipsea) C:\Users\malpka\Desktop\venv-bipsea>bipsea --help
Traceback (most recent call last):
  File "C:\Users\malpka\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\malpka\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\malpka\Desktop\venv-bipsea\Scripts\bipsea.exe\__main__.py", line 4, in <module>
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\bipsea\bipsea.py", line 9, in <module>
    from .bip32 import to_master_key
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\bipsea\bip32.py", line 15, in <module>
    from .bip32types import VERSIONS, ExtendedKey
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\bipsea\bip32types.py", line 8, in <module>
    from .util import LOGGER_NAME
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\bipsea\util.py", line 15, in <module>
    POETRY = Factory().create_poetry()
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\poetry\factory.py", line 58, in create_poetry
    base_poetry = super().create_poetry(cwd=cwd, with_groups=with_groups)
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\poetry\core\factory.py", line 50, in create_poetry
    poetry_file = self.locate(cwd)
  File "C:\Users\malpka\Desktop\venv-bipsea\lib\site-packages\poetry\core\factory.py", line 507, in locate
    raise RuntimeError(
RuntimeError: Poetry could not find a pyproject.toml file in C:\Users\malpka\Desktop\venv-bipsea or its parents

```

**Comment by malpka** [comment]
It is possible that not venv is the problem, but pip package itself.

**Comment by akarve** [comment]
~~Interesting. What happens if you clone this repo then run the following?~~
```
make test-dist
```

> Update: I can repro this in envs without poetry.

**Comment by akarve** [comment]
Should be fixed in 2.0.1. Lmk if you bump into anything else.

**Comment by malpka** [comment]
> Should be fixed in 2.0.1. Lmk if you bump into anything else.

Fixed, thank you
